gem/spec/configure_earlgrey_spec.rb

```diff
@@ -22,8 +22,22 @@
     expect(EarlGrey.carthage).to eq(true)
   end
 
-  it 'configures for cocoapods' do
-    diff_project cocoapods_after, %w[install -t AutoEarlGrey --no-carthage]
+  it 'configures for cocoapods with default swift version' do
+    diff_project cocoapods_after_default, %w[install -t AutoEarlGrey --no-carthage]
+    expect(EarlGrey.swift).to eq(true)
+    expect(EarlGrey.carthage).to eq(false)
+  end
+
+  it 'configures for cocoapods with Swift 2.2' do
```

Comment:
technically this works however we'll potentially end up with 6 fixture projects

2 (cocoapods + carthage) * 3 (swift default, swift 2, swift 3) = 6 

I'd rather keep one baseline for both cocoapods & carthage then dynamically modify the data in memory with the correct path. The only change is the file path so I think this is reasonable.